app/views/posts/id.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Here is a list of posts for a given user</h1>
```

Comment:
- [ ] Please change the file name from `id.html.erb` to `index.html.erb` to follow to the naming convention